infrastructure/models.py

```diff
@@ -50,22 +66,40 @@ def __str__(self):
 
 
 class ProjectStatus:
+    NULL = 0
     ANNOUNCED = 1
     PREPATORY = 2
     STARTED = 3
     UNDER_CONSTRUCTION = 4
     COMPLETED = 5
     CANCELLED = 6
+    ACTIVE = 7
+    PARTIALLY_ACTIVE = 8
+    INACTIVE = 9
 
     STATUSES = (
+        (NULL, 'Null'),
         (ANNOUNCED, 'Announced/Under Negotiation'),
         (PREPATORY, 'Preparatory Works'),
         (STARTED, 'Started'),
+        (ACTIVE, 'Active'),
+        (PARTIALLY_ACTIVE, 'Partially Active'),
+        (INACTIVE, 'Inactive'),
         (UNDER_CONSTRUCTION, 'Under Construction'),
         (COMPLETED, 'Completed'),
         (CANCELLED, 'Cancelled'),
     )
 
+class ProjectPlantUnits:
+    MEGAWATTHOUR = 0
+    MEGAWATT = 1
+    TONSPERANNUM = 2
+
+    UNITS = (
+        (MEGAWATTHOUR, 'MWh'),
+        (MEGAWATT, 'MW'),
+        (TONSPERANNUM, 'Tons per annum')
```

Comment:
In the CSV sheet this is "Tonnes per annum", which is [a little different](https://blog.harwardcommunications.com/2012/01/23/the-difference-between-ton-and-tonne/) than "Tons per annum"